src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerIndividualValidator.java

```diff
@@ -96,7 +96,7 @@ private boolean validateNationality(String nationality, Errors errors, String lo
     }
 
     private Errors validateAddress(String addressField, AddressDto addressDto, Errors errors, String loggingContext) {
-        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.ENTITY_FIELD, addressField);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.BENEFICIAL_OWNERS_INDIVIDUAL_FIELD, addressField);
```

Comment:
As the unit tests didn't catch this bug, we'll need to update the unit test to check that the correct qualified name is passed into the addressDtoValidator

Reply:
Updated tests for both officer blocks and for the other blocks as well.